src/Cli.Tests/EnvironmentTests.cs

```diff
@@ -99,13 +99,22 @@ public void TestPrecedenceOfEnvironmentFileOverExistingVariables()
                     ""envValue"": ""DEVELOPMENT"",
                     ""hostingEnvValue"": ""PHOENIX_TEST""
                 }"),
-            JObject.Parse(resolvedJson)));
+            JObject.Parse(resolvedJson)), "JSON resolved with environment variable correctly");
 
         // Removing the .env file it will then use the value of system environment variable.
-        Environment.SetEnvironmentVariable(envVariableName, "TEST");
-        File.Delete("test.env");
-        DotNetEnv.Env.Load("test.env");
-        Assert.AreEqual("TEST", Environment.GetEnvironmentVariable(envVariableName));
+
+    }
+
+    /// <summary>
+    /// Test to verify that if .env file is not present then existing system variables will be used.
+    /// </summary>
+    [TestMethod]
+    public void TestSystemEnvironmentVariableIsUsedInAbsenceOfEnvironmentFile()
+    {
+        Environment.SetEnvironmentVariable(TEST_ENV_VARIABLE, "TEST");
+        Assert.IsFalse(File.Exists("test.env"));
+        DotNetEnv.Env.Load("test.env"); // No error is thrown
```

Comment:
How was this resolved? I dont think we should call DotNetEnv.Env.Load in the test. Since that would mean we are testing this externally - not the actual usage of Load in DAB. Do we need to add such a test at all?

Reply:
We have tests that verifies DotNetEnv.Env.Load internally. [Line:125(Environment)](https://github.com/Azure/data-api-builder/blob/029ad357bd83b547d1c32f38f8f0ae4ae8adfe9c/src/Cli.Tests/EnvironmentTests.cs#L125)

we have some external tests as well to just verify the method ParseConfigJsonAndReplaceEnvVariables as calling this directly won't load the variables from .env file.

Loading from .env file happens at the initialization step of DAB CLI, so that every cli command is synced with the environment variables.